boot/bootutil/src/bootutil_priv.h

```diff
@@ -62,16 +62,24 @@ struct flash_area;
 /** Number of image slots in flash; currently limited to two. */
 #define BOOT_NUM_SLOTS                  2
 
-#if !defined(MCUBOOT_OVERWRITE_ONLY)
+#if defined(MCUBOOT_OVERWRITE_ONLY) && defined(MCUBOOT_SWAP_MOVE)
+#error "Please enable only one of MCUBOOT_OVERWRITE_ONLY or MCUBOOT_SWAP_MOVE"
+#endif
+
+#if !defined(MCUBOOT_OVERWRITE_ONLY) && !defined(MCUBOOT_SWAP_MOVE)
```

Comment:
I prefer to state explicitly how to :
do swap using scratch algorithm  or -> MCUBOOT_SWAP_USING_SCRATCH
do swap using image sector shift algorithm -> MCUBOOT_SWAP_USING_MOVE
do not swap ->MCUBOOT_OVERWRITE_ONLY

Reply:
That is a good idea actually, I was unable to make my mind about the best naming to use!

Reply:
Done!